test/chtest-make-auto

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+fatal () {
+    printf "error: %s\n" "$1" >&2
+    exit 1
+}
+
+tag_from_path () {
+    parent_dir=$(dirname "$path_")
```

Comment:
I would call this `dir` because `parent` makes me ask "parent of what". I think only directories have parents. Files have enclosing or containing directories.